src/testing/CMakeLists.txt

```diff
@@ -36,6 +37,7 @@ foreach(dim RANGE 1 3)
     string(CONCAT OptimizationLib Optimization_${dim}D)
     string(CONCAT LinearSolverLib LinearSolver)
     string(CONCAT GridRefinementLib GridRefinement_${dim}D)
+    string(CONCAT PODLib POD_${dim}D)
```

Comment:
could use full name for POD here?

Reply:
As we discussed, I renamed pod.cpp and pod.h to proper_orthogonal_decomposition.cpp and proper_orthogonal_decompostition.h.